src/constants.ts

```diff
@@ -73,7 +74,70 @@ export const questionTakenFileName = {
     "Looks like there's already a file with that name. What would you like to name your file?",
 };
 
-export const DOCS_URL = 'https://airfoil.teamairship.com';
+export const questionValidHookName = {
+  type: 'input',
+  name: 'name',
+  message:
+    'When naming a hook, make sure the hook\'s name follows the pattern useHookName (camelcase starting with "use"). What would you like to name your hook?',
+};
+
+// ------------------------
+// ----- DESCRIPTIONS -----
+// ------------------------
+export const HELP_DESCRIPTION_CMD_ADD = `airfoil add [adr | appcenter | component | env | hook | keystore ]
```

Comment:
Help descriptions already exist in `./src/help-descriptions` - could we move the additional one there?

Reply:
See: https://github.com/teamairship/airfoil/tree/develop/src/help-descriptions